src/m365/spo/commands/list/list-get.ts

```diff
@@ -49,8 +51,10 @@ class SpoListGetCommand extends SpoCommand {
       requestUrl = `${args.options.webUrl}/_api/web/lists/GetByTitle('${encodeURIComponent(args.options.title as string)}')`;
     }
 
+    const propertiesSelect: string = args.options.properties ? `?$select=${encodeURIComponent(args.options.properties)}` : ``;
+
     const requestOptions: any = {
-      url: requestUrl,
+      url: `${requestUrl}${propertiesSelect}`,
```

Comment:
Since we just need to add these two strings, let's change it to `requestUrl + propertiesSelect` to simplify it and ensure that nothing else creeps in between.